hub/hub_adc.go

```diff
@@ -245,6 +245,40 @@ func (h *Hub) adcStageIdentity(peer *adcPeer) error {
 
 	st := h.Stats()
 
+	isRegistered, err := h.isRegisteredUserADC(peer.Name())
+	if err != nil {
+		return err
+	}
+	if isRegistered {
+		// give the user a minute to enter a password
+		deadline = time.Now().Add(time.Minute)
+		//some bytes for check password
+		data := []byte("JJWEPPPLCA3PF2ZCRRYO3333")
```

Comment:
This is a base32 representation, but it will be better to have raw bytes here, and the `GetPassword` command will encode it into base32.